lib/states/menu_state.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module States
+  class MenuState < BaseState
+    START_COMMAND = I18n.t('menu_state.start_command')
+    RULES_COMMAND = I18n.t('menu_state.rules_command')
+    STATS_COMMAND = I18n.t('menu_state.stats_command')
+
+    def interact
+      puts(I18n.t('menu_state.introduction'))
+      puts(I18n.t('menu_state.game_menu_options', start: START_COMMAND, rules: RULES_COMMAND, stats: STATS_COMMAND,
+                                                  exit: EXIT_COMMAND))
+      command = user_input
+      context.change_state_to(manage_command(command))
+    end
+
+    private
+
+    def manage_command(command)
+      case command
+      when START_COMMAND then return States::RegistrationState.new
+      when RULES_COMMAND then puts(I18n.t('menu_state.rules'))
+      when STATS_COMMAND then put_statistic
+      else
+        puts I18n.t('base_state.wrong_command')
+      end
+
+      self
+    end
+
+    def put_statistic
```

Comment:
What does it mean statistic? And why put? Think about what this method does. If it's printing something, it might be better to call it print_statistics.